app/src/main/java/edu/uc/reedws/musiclink/dto/SongDTO.kt

```diff
@@ -1,3 +1,5 @@
 package edu.uc.reedws.musiclink.dto
 
-data class SongDTO(var title: String, var album: String, var artist: String)
\ No newline at end of file
+data class SongDTO(var title: String, var album: String, var artist: String) {
+    override fun toString() = "$title $album $artist"
+}
```

Comment:
It is always a good practice to have the ability to return a string of variables inside the data class to check the values inside the data class. I learned this from professor, but I have also looked it up in kotlin coding conventions. I am returning all the values from the data class, but you can tailored the string according to your need in the future.

Kotlin coding convention for data class: https://kotlinlang.org/docs/reference/data-classes.html